schemas/storage.sql

```diff
@@ -374,3 +374,51 @@ WHERE StarRecords.StorageSystemID = StorageSystems.id
   AND StarRecords.GroupID = Groups.id
   AND StarRecords.SubGroupID = SubGroups.id
   AND StarRecords.RoleID = Roles.id;
+
+-- -----------------------------------------------------------------------------
+-- Summaries
+DROP TABLE IF EXISTS DaySummaries;
+CREATE TABLE DaySummaries (
+  Site                        VARCHAR(255),
```

Comment:
See https://github.com/apel/apel/pull/229/files#r1188726030, if so - should all the VARCHAR rows be INTs instead? Like this:

```suggestion
  SiteID                        INT NOT NULL),
```

Reply:
@Sae126V, can you add additional commits to the PR to change this (and https://github.com/apel/apel/pull/229/files#r1188726030)? The procedures this PR has added will likely need to be updated.

There are already lookup functions in the storage schema which should handle the lookup of a names to IDs (i.e. `Site` to `SiteID`).

This might have some knock on effects for https://github.com/apel/apel/pull/315.